code/modules/clothing/spacesuits/rig/suits/combat.dm

```diff
@@ -26,4 +25,59 @@
 		/obj/item/rig_module/power_sink,
 		/obj/item/rig_module/electrowarfare_suite,
 		/obj/item/rig_module/chem_dispenser/combat
+		)
+
+/obj/item/weapon/rig/combat/empty
+	initial_modules = list(
+		/obj/item/rig_module/ai_container,
+		/obj/item/rig_module/electrowarfare_suite,
+		)
+
+/obj/item/weapon/rig/military
+	name = "military hardsuit control module"
+	desc = "An austere hardsuit used by paramilitary groups and real soldiers alike."
+	icon_state = "military_rig"
+	suit_type = "military hardsuit"
+	armor = list(melee = 80, bullet = 70, laser = 55, energy = 15, bomb = 80, bio = 100, rad = 30)
+	slowdown = 1
+	offline_slowdown = 3
+	offline_vision_restriction = 1
+	allowed = list(/obj/item/device/flashlight, /obj/item/weapon/tank,/obj/item/ammo_magazine,/obj/item/ammo_casing,/obj/item/weapon/handcuffs,/obj/item/device/t_scanner, /obj/item/weapon/rcd, /obj/item/weapon/tool/crowbar, \
+	/obj/item/weapon/tool/screwdriver, /obj/item/weapon/weldingtool, /obj/item/weapon/tool/wirecutters, /obj/item/weapon/tool/wrench, /obj/item/device/multitool, \
+	/obj/item/device/radio, /obj/item/device/analyzer,/obj/item/weapon/storage/briefcase/inflatable, /obj/item/weapon/melee/baton, /obj/item/weapon/gun, \
+	/obj/item/weapon/storage/firstaid, /obj/item/weapon/reagent_containers/hypospray, /obj/item/roller, /obj/item/device/suit_cooling_unit)
+
+	chest_type = /obj/item/clothing/suit/space/rig/military
+	helm_type = /obj/item/clothing/head/helmet/space/rig/military
+	boot_type = /obj/item/clothing/shoes/magboots/rig/military
+	glove_type = /obj/item/clothing/gloves/rig/military
```

Comment:
This should be `/obj/item/clothing/gloves/gauntlets/rig/military`, every rig glove is a subtype of gauntlet so it can go over normal gloves.